warrior/Framework/Utils/data_Utils.py

```diff
@@ -1446,8 +1458,15 @@ def subst_var_patterns_by_prefix(raw_value, start_pattern="${",
                                     str(raw_value[k]).replace(
                                         search_obj.group(), 'None'))
                     except SyntaxError:
-                        print_error("Syntax Error - " +
-                                    error_msg2.format(string, value))
+                        from WarriorCore.Classes.testcase_utils_class import TestcaseUtils
+                        tuc_obj = TestcaseUtils()
+                        print_info("Checking after removing the illegal characters")
```

Comment:
Please provide a more detailed message
"Cannot convert value: {} into correct format, removing non-printable characters, will attempt conversion again"

Reply:
Hi @navvrnd 
The code looks good, I run the new testcase and verified it works
however cli_proj is failing and it shouldn't
please refer to the internal PR for more info